x-pack/plugins/timelines/common/types/timeline/actions/index.ts

```diff
@@ -53,15 +53,30 @@ export type OnUpdateAlertStatusSuccess = (
 ) => void;
 export type OnUpdateAlertStatusError = (status: AlertStatus, error: Error) => void;
 
-export interface StatusBulkActionsProps {
+export interface CustomBulkAction {
+  key: string;
+  label: string;
+  disableOnQuery?: boolean;
+  disabledLabel?: string;
+  onClick: (items?: TimelineItem[]) => void;
+  ['data-test-subj']?: string;
+}
+
+export type CustomBulkActionProp = Omit<CustomBulkAction, 'onClick'> & {
+  onClick: (eventIds: string[]) => void;
```

Comment:
Shouldn't this be `onClick: (items?: TimelineItem[]) => void;`? . Do I miss something?

Reply:
no. This returns eventId strings. You can't generate an array of timeline items here.